packages/initiatives/src/migrations/apply-schema.ts

```diff
@@ -0,0 +1,85 @@
+import { IInitiativeModel } from "@esri/hub-common";
+import { getProp, cloneObject } from "@esri/hub-common";
+
+/* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
```

Comment:
nitpick, but the copyright belongs way up top.

Reply:
fixed